compiler/test/stdlib/uint32.test.gr

```diff
@@ -0,0 +1,88 @@
+module Uint32Test
+
+let (===) = (==)
```

Comment:
I thought it would just make it more convenient; in either case the `(==)` function in Uint32 is tested in this file which would require something like `Uint32.(==)(a, b)` which also feels awkward (unless there is a way to use it as an infix operator here that I don't know about)

Reply:
We could separate `Pervasive.(==)` and `Uint32.(==)` tests using blocks and use `from Uint32 use *` inside the blocks, right?

Reply:
True; I think that would still require `==` to be aliased for the block with `use *` though so maybe I could just import the functions used individually and alias `Uint32.(==)` as `eq` or something

Reply:
Since other comparison operators were also involved I just ended up grouping them at the end and adding a `from` statement specifically for comparison operators after the other tests